vnext/local-cli/generator-windows/templates/cpp/src/App.xaml

```diff
@@ -1,8 +1,8 @@
-﻿<react:ReactApplication
-    x:Class="<%=ns%>.App"
+﻿<react:ReactApplication 
+    x:Class="{{ namespace }}.App"
```

Comment:
I think this might be wrong - this namespace should be in the x.y.z format, whereas in the cpp file it must be in the x::y::z format

Reply:
probably pre-existing but yeah

Reply:
Since pre-existing I'll investigate and address if needed in separate pr.

Reply:
I assumed you were saying that it should be colon separated because today it is dot-separated... The namespace variable contains dot's. The issue is with the cpp file :) In my next PR I'll add proper naming and fix the CPP case.

Reply:
@dannyvv Filed #5144